apps/web/src/components/CopyToClipboard.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from "react";
+import { CheckIcon } from "@heroicons/react/24/outline";
+
+import Copy from "~/icons/copy.svg";
+import { Tooltip } from "./Tooltip";
+
+export function CopyToClipboard({
+  label,
+  value,
+}: {
+  label: string;
+  value: string;
+}) {
+  const [isCopied, setCopied] = useState(false);
+  const [isHover, setIsHover] = useState(false);
+
+  return (
+    <button
```

Comment:
We can keep using this button until we merge the `Button` refactoring PR.

Maybe we can add a `// TODO:` here to change this after merging that PR 🤔

Reply:
[8ba525d](https://github.com/Blobscan/blobscan/pull/478/commits/8ba525d7a08d43f5f7e0b9801b593564346f3c67)